content/ch-gates/multiple-qubits-entangled-states.ipynb

```diff
@@ -2374,6 +2374,19 @@
     "plot_histogram(results)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The other three Bell's states are,\n",
```

Comment:
I think the comma at the end of the line could be removed or replaced with a colon, what do you think?